src/swap/hooks/useFromTo.ts

```diff
@@ -28,6 +31,7 @@ export const useFromTo = (address?: Address) => {
     loading: fromLoading,
     setLoading: setFromLoading,
     error: fromTokenBalanceError,
+    response: fromTokenResponse,
```

Comment:
Ok dope, can you give it an alphabetical order clean up, for the `from` and `to`. So it's easier to follow.

Reply:
cleaned up/ordered alphabetically